eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/persistence/DbsnpVariantsWrapper.java

```diff
@@ -69,6 +72,14 @@ public void setOperations(List<DbsnpSubmittedVariantOperationEntity> operations)
         this.operations = operations;
     }
 
+    public void addOperation(DbsnpSubmittedVariantOperationEntity operation) {
```

Comment:
This method is currently unused. Depending on whether the operations are added to an existing collection, as described in previous comments, that could change.